app/code/Magento/Catalog/Model/CategoryList.php

```diff
@@ -31,15 +30,11 @@ class CategoryList implements CategoryListInterface
     private $extensionAttributesJoinProcessor;
 
     /**
-     * @var CategorySearchResultsInterfaceFactory
+     * @var CategorySearchResultsInterfaceFac
+     tory
```

Comment:
I cannot find this line break in the file when editing it. I have no idea what is causing this.